src/InstagramClone/InstagramClone.Web/Controllers/ImageController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using InstagramClone.Web.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace InstagramClone.Web.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class ImageController : ControllerBase
+    {
+        private static List<Image> _images = new List<Image>();
+
+        [HttpPost("upload")]
+        public async Task<IActionResult> Upload([FromForm] string title, [FromForm] string uploader, [FromForm] string description, [FromForm] IFormFile file)
```

Comment:
Itt mi lenne, ha egy DTO-ként érkezne már be? Ki tudnánk használni, hogy a .NET Core automatikusan rá map-elné a megfelelő DTO-ra Model Binding-al.